src/main/java/use_cases/UserManager.java

```diff
@@ -2,176 +2,199 @@
+     * Update User's password into username given
+     *
+     * @param user The user that will get its username updated
+     * @param new_password The new password that the user will take
      */
     public void updatePassword(User user, String new_password) {
         user.setPassword(new_password);
     }
 
     /**
      * Update User's bio into bio given
+     *
+     * @param user The user that will get its username updated
+     * @param bio The new bio that the user will take
      */
-    public void update_bio(User user, String bio) {
+    public void updateBio(User user, String bio) {
         user.setBio(bio);
     }
 
     /**
      * Add 1 like to the cuisine category that User liked
+     *
+     * @param user The current logged-in user
+     * @param cuisineCategory the cuisine category of the post that the user liked
      */
     public void addLike(User user, String cuisineCategory) {
         int currentLikes = user.getLikeHistory().get(cuisineCategory);
         currentLikes = currentLikes + 1;
         user.getLikeHistory().put(cuisineCategory, currentLikes);
-//        Transforming this function into the one line below makes it less readable
-//        user.getLike_history().put(cuisine_category, user.getLike_history().get(cuisine_category) + 1);
     }
 
     /**
-     * follow User want_to_follow, adding want_to_follow into the logged-in user's following list
+     * Follow a user that the currently logged-in user wishes to follow
+     *
+     * @param currentUser the current logged-in user
+     * @param wantToFollow the user that the current logged-in user wishes to follow
+     * @return true if user wantToFollow is successfully added to user's following list, false otherwise
      */
     public boolean followUser(User currentUser, User wantToFollow) {
-        /**
-         * Return false if user is already followed, true otherwise.
-         */
         // check whether User want_to_follow is already followed by current_user
         if (currentUser.getFollowing().contains(wantToFollow)) {
             return false;
-        }
-        else {
+        } else {
             // update current_user's following list: following
             ArrayList<User> updatedFollowingList = new ArrayList<>();
             updatedFollowingList.add(wantToFollow);
-            for (User user: currentUser.getFollowing()) {
-                updatedFollowingList.add(user);
-            }
+            updatedFollowingList.addAll(currentUser.getFollowing());
             currentUser.setFollowing(updatedFollowingList);
             return true;
         }
     }
 
     /**
-     * unfollow User want_to_unfollow, removing want_to_unfollow from the logged-in user's following list,
-     * return false if want_to_unfollow is not followed to begin with, true if action done successfully
+     * unfollow User want_to_unfollow, removing want_to_unfollow from the logged-in user's following list
```

Comment:
Could we rename the variable names in the comment to use camelCase?

Reply:
Or just put a space between

Reply:
yeah sounds good